source/dstats/random.d

```diff
@@ -381,7 +381,7 @@ unittest {
     writeln("\tStdev Expected:  ", sqrt(2 * df), "  Observed:  ", stdev(observ));
     writeln("\tKurtosis Expected:  ", 12 / df, "  Observed:  ", kurtosis(observ));
     writeln("\tSkewness Expected:  ", sqrt(8 / df), "  Observed:  ", skewness(observ));
-    delete observ;
+    destroy(observ);
```

Comment:
`destroy` doesn't deallocate memory, it just calls destructors/finalizers. See alternative https://github.com/DlangScience/dstats/pull/42.